cmd/resource/status/cmd/util.go

```diff
@@ -55,8 +58,20 @@ func (f *CaptureIdentifiersFilter) Filter(slice []*yaml.RNode) ([]*yaml.RNode, e
 		if err != nil {
 			return nil, err
 		}
+		// TODO(mortent): Update kyaml library
 		id := meta.GetIdentifier()
-		f.Identifiers = append(f.Identifiers, &id)
+		gv, err := schema.ParseGroupVersion(id.APIVersion)
+		if err != nil {
+			return nil, err
+		}
+		f.Identifiers = append(f.Identifiers, wait.ResourceIdentifier{
+			Name:      id.Name,
+			Namespace: id.Namespace,
```

Comment:
I think we should pass id.Namespace to be "default" in case it is empty string. The current output has empty namespace. It should output "default" is what I thing even if the input yaml doesn't have a namespace.

![image](https://user-images.githubusercontent.com/58999035/72281472-7c794680-35ef-11ea-8789-f4d17f269eb4.png)

And why are we getting additional output for "/"? Shouldn't that be filtered from the output.

Reply:
And can you choose another color please :D I suggest black with status being red/green.

Reply:
So the challenge with the namespace is that we can't decide if a resource is namespace-scoped or cluster-scoped just by reading the manifest from the file. If it is namespace-scoped, it would make sense to set the namespace to "default", but it would be strange for cluster-scoped resources. We can discuss this but I would like to handle it in a separate PR.

The output color needs to change based on the background color of the terminal. This means we need to look into if we can use ascii escape sequences to read back the background color. Let's handle this one too in a different PR.